### PR TITLE
docs: update curation page with arbitrum one changes

### PR DESCRIPTION
**Pull request by tmigone**

### CODE REVIEW

File: website/pages/en/network/curating.mdx [hunk 1]
```diff
@@ -81,17 +87,20 @@ Finding high-quality subgraphs is a complex task, but it can be approached in ma
...
 - Curators should also understand the metrics that are available through The Graph Explorer. Metrics like past query volume and who the subgraph developer is can help determine whether or not a subgraph is worth signalling on.
 
-### 3. What’s the cost of updating a subgraph?
+### 3. What’s the cost of upgrading a subgraph?
 
-Migrating your curation shares to a new subgraph version incurs a curation tax of 1%. Curators can choose to subscribe to the newest version of a subgraph. When curation shares get auto-migrated to a new version, Curators will also pay half curation tax, ie. 0.5%, because updating subgraphs is an on-chain action that costs gas.
+Migrating your curation shares to a new subgraph version incurs a curation tax of 1%. Curators can choose to subscribe to the newest version of a subgraph. When curator shares get auto-migrated to a new version, Curators will also pay half curation tax, ie. 0.5%, because upgrading subgraphs is an on-chain action that costs gas.
 
-### 4. How often can I update my subgraph?
+### 4. How often can I upgrade my subgraph?
```
Comment (azf20) [inline website/pages/en/network/curating.mdx:94]: I think we use `update` for new versions ("upgrade" is used to reference moving to the network)

File: website/pages/en/network/curating.mdx [hunk 2]
```diff
@@ -2,71 +2,77 @@
...
-
-![Signaling diagram](/img/curator-signaling.png)
+Curators make The Graph network efficient and signaling is the process that curators use to let Indexers know that a subgraph is good to index, where GRT is added to a bonding curve for a subgraph. Indexers can inherently trust the signal from a curator because upon signaling, curators mint a curation share for the subgraph, entitling them to a portion of future query fees that the subgraph drives. Curator signal is represented as ERC20 tokens called Graph Curation Shares (GCS). Curators that want to earn more query fees should signal their GRT to subgraphs that they predict will generate a strong flow of fees to the network. Curators cannot be slashed for bad behavior, but there is a deposit tax on Curators to disincentivize poor decision-making that could harm the integrity of the network. Curators also earn fewer query fees if they choose to curate on a low-quality Subgraph since there will be fewer queries to process or fewer Indexers to process those queries.
```
Comment (azf20) [inline website/pages/en/network/curating.mdx:37]: I am not sure what "inherently trust" means here (I know it was in the original text)

File: website/pages/en/network/curating.mdx [hunk 3]
```diff
@@ -2,71 +2,77 @@
...
+The Graph Network's original deployment on Ethereum uses bonding curves to determine what the price of shares is: **the price of each subgraph share increases with each token invested** and the **price of each share decreases with each token sold.** This means that curating puts your principal at risk, since it's not guaranteed you can sell your shares and get back your original investment.
 
-In the case of The Graph, [Bancor’s implementation of a bonding curve formula](https://drive.google.com/file/d/0B3HPNP-GDn7aRkVaV3dkVl9NS2M/view?resourcekey=0-mbIgrdd0B9H8dPNRaeB_TA) is leveraged.
+On the Arbitrum deployment curating subgraphs becomes significantly more advantageous. The bonding curves are "flattened", their effect is nullified meaning no curator will be able to realize gains at the expense of others. This allows Curators to signal or unsignal on subgraphs at any given time, at a consistent cost with very limited risk.
```
Comment (azf20) [inline website/pages/en/network/curating.mdx:15]: I prefer "simpler" to "more advantageous"

**Review by azf20 — APPROVED** [review]
Thanks! Some small comments

File: website/pages/en/network/curating.mdx (hunk 1 above)
Comment (tmigone) [reply website/pages/en/network/curating.mdx:94]: good idea to change that

File: website/pages/en/network/curating.mdx (hunk 2 above)
Comment (tmigone) [reply website/pages/en/network/curating.mdx:37]: yeah not sure either, removed the `inherently`

**Pull request merged**